### DISCUSSION TITLE
Taskbar not hidden on some fullscreen games like world of warcraft

### DISCUSSION THREAD

**Discussion by PlanetRageTV** [lawand-dot-io/taskbar#102] [Feature requests]
Hi, love the taskbar but when playing a game like world of warcraft the taskbar stays in the foreground in front of the game. I can hide it with the gesture but whenever my mouse comes close to the bottom (which happens frequently in games) it pops up again. Any way it can detect 3D games in Fullscreen and hide itself?

**Comment by lawand-dot-io** [comment]
@PlanetRageTV Hello. This is a bug. World of warcraft is supposed to be detected as fullscreen and Taskbar is supposed to be hidden while playing it in fullscreen. This bug doesn't happen for example if you open a Youtube video in the browser and double click to. fullscreen it. What app are you using to launch the game? Are you using Steam?

Is this bug happening with other games or other apps?

A temporary solution is to right click Taskbar and select the "Hide" option which keeps it hidden even when the mouse is at the bottom of the screen. If you do that, when you exit the game and want to show Taskbar again, you have to use the Taskbar icon in the menubar at the top of the screen and select the "Show" option. Thanks

**Comment by lawand-dot-io** [comment]
This is moved into https://github.com/lawand-dot-io/taskbar/issues/180
